build.gradle

```diff
@@ -76,6 +76,9 @@ def configureKotlinJvmPlatform(configuration) {
     configuration.attributes.attribute(KotlinPlatformType.attribute, KotlinPlatformType.jvm)
 }
 
+// Configure subprojects with Kotlin sources
+apply plugin: "configure-compilation-conventions"
```

Comment:
I'm okay with this particular kludge.

What puzzles me is the general debuggability, i.e. consider the same situation happens to an external user:

>When we apply the atomicfu plugin too late in the configuration -- the AtomicfuKotlinGradleSubplugin$isApplicable may be called earlier, when the properties are not set yet and the compiler plugin is not applied.

What are the option for them to debug it and understand what's going on? Are there better options?

Reply:
imho this case is nothing to do with Atomicfu plugin as it is not even present when core project evaluates. This is more Gradle edge-case. BTW why do you need `evaluationDependsOn` for core project? :thinking: